.travis.yml

```diff
@@ -2,8 +2,5 @@ language: python
 python:
   - "2.6"
   - "3.5"
-install:
-  - "pip install --upgrade pip"
-  - "pip install pep8"
-  - "pip install -r requirements.txt"
-script: "pep8 --ignore=E402 --max-line-length=160 ."
+install: make test-setup
```

Comment:
I just realized: this `make` is superfluous. There is no target `make`. (same on the following line)

Reply:
Could you explain? I think we need `make` here.

Reply:
Sorry, i was wrong. the line looked too much like Makefile to me to realize it is the travis config.
Complaint withdrawn. 🙈